dask_jobqueue/tests/test_slurm.py

```diff
@@ -6,9 +6,12 @@
 
 from dask_jobqueue import SLURMCluster
 
+pytestmark = pytest.mark.env("slurm")
```

Comment:
Do you need this here? IIRC the consensus was to add `pytest.mark.env` only for decorating functions that need a real cluster to run.

The same thing happens in `test_pbs.py`.